tests/parse-legacy-file.bats

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env bats
+
+# SPDX-License-Identifier: MIT
+# SPDX-FileCopyrightText: Copyright (c) 2017 Jack Henry & Associates
+
+setup() {
+  ASDF_OPENTOFU="$(dirname "$BATS_TEST_DIRNAME")"
+  PARSE_LEGACY_FILE="${ASDF_OPENTOFU}/bin/parse-legacy-file"
+}
+
+@test "supports legacy opentofu version 'required_version' with strict equality" {
+  local -r expected_opentofu_version=0.13.7
+  local -r tmpdir="$(mktemp -d)"
+  local -r version_file="${tmpdir}/main.tofu"
+  cat <<EOF >"${version_file}"
+terraform {
```

Comment:
This one is missing

```suggestion
opentofu {
```

Reply:
@hinricht we can merge after that one is fixed :+1:

Reply:
Ok, I changed all config blocks from `opentofu {` to `terraform {`.